client/src/components/MultiJurisdictionAudit/AASetup/Review/index.tsx

```diff
@@ -53,7 +53,19 @@ const Review: React.FC<IProps> = ({ prevStage, locked, refresh }: IProps) => {
   const [sampleSizes, setSampleSizes] = useState<IFormOptions>({})
   const history = useHistory()
   const [isConfirmDialogOpen, setIsConfirmDialogOpen] = useState(false)
-  const [sampleSizeOptions, uploadSampleSizes] = useSampleSizes(electionId)
+
+  const talliesUploadsCompleted =
+    !!jurisdictions.length &&
+    jurisdictions.every(
+      j =>
+        j.batchTallies &&
+        j.batchTallies.processing &&
+        j.batchTallies.processing.status === FileProcessingStatus.PROCESSED
+    )
+  const [sampleSizeOptions, uploadSampleSizes] = useSampleSizes(
+    electionId,
+    auditType === 'BALLOT_POLLING' || talliesUploadsCompleted // only fetch sample sizes for ballot polling audits or if all tallies files are uploaded
+  )
```

Comment:
ok i should have thought of this earlier but the idea just came to me. something seems a little weird to me about having to tell the `useSampleSizes` hook whether it should fetch or not. instead, what if you pulled out the sample size form into its own component that calls the `useSampleSizes` hook? then you could conditionally render the component instead of putting the condition into the hook

Reply:
@jonahkagan Yeah, that could conceivably work, but again that runs into issues with the submit button. If you want to show a disabled submit button instead of nothing, we'll need to have them both in this component.

Reply:
oh, right. maybe just the form section then?

Reply:
@jonahkagan I just looked, but we need to declare the initial values outside of the form, and that requires the sample size options. So extracting out the `useSampleSizes` hook elsewhere would not work without adding complexity. I think passing in the toggle for whether the hook should fetch or not is the simplest solution at this point.

Reply:
ah i see what you mean